### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by sunny-panchal**
This PR:
- Adds the code of conduct from https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
- Adds a minimal CONTRIBUTING.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (nahuakang) [inline CODE_OF_CONDUCT.md:64]: This contact method still needs to be provided :smile:

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].
+
+Community Impact Guidelines were inspired by 
+[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
+
+For answers to common questions about this code of conduct, see the FAQ at
+[https://www.contributor-covenant.org/faq][FAQ]. Translations are available 
+at [https://www.contributor-covenant.org/translations][translations].
+
+[homepage]: https://www.contributor-covenant.org
```
Comment (nahuakang) [inline CODE_OF_CONDUCT.md:129]: Should the homepage also be theirs or should it be ours?

**Review by nahuakang — COMMENTED** [review]
Looks good mostly :+1: Thanks for finding a good CoC. Left 2 comments.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sunny-panchal) [reply CODE_OF_CONDUCT.md:64]: Yeah, still trying to figure out which one we should put

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (sunny-panchal) [reply CODE_OF_CONDUCT.md:129]: It's meant to be a link to theirs in this case

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sunny-panchal) [inline CODE_OF_CONDUCT.md:64]:
```suggestion
opensource@twentybn.com.
```

**Review by guillaumebrg — APPROVED** [review]

**Pull request merged**